sycl/test-e2e/Basic/image/srgba-read.cpp

```diff
@@ -1,4 +1,5 @@
 // REQUIRES: aspect-ext_oneapi_srgb, aspect-ext_intel_legacy_image
+// UNSUPPORTED: level_zero
```

Comment:
Where did it fail? `aspect-ext_oneapi_srgb` above should have been enough to skip the test.

Reply:
@aelovikov-intel : this aborts on following line

https://github.com/intel/llvm/blob/d1f67ca228f222fb1cbe7c8cfa80e1103d7af133/sycl/plugins/level_zero/pi_level_zero.cpp#L3505

dont know if aspect-ext_oneapi_srgb  should catch it, but for the changes we are adding to use the unified-runtime, we are not seeing this skipped. What "aspect-ext_oneapi_srgb"  should do?

Reply:
Good point! @jandres742 : were we somehow reporting `aspect-ext_oneapi_srgb` as supported with UR?

Reply:
@smaslov-intel : how are we checking that support to start with in L0 plugin?

Reply:
> What "aspect-ext_oneapi_srgb" should do?

What aspects does `sycl-ls --verbose` report in your configuration?

Reply:
thanks @smaslov-intel , @aelovikov-intel . I found a casting error in the UR path, which was making to incorrectly report as true for SRGBA. Closing.